drivers/hmis/app/models/hmis/form/assessment_detail.rb

```diff
@@ -9,6 +9,12 @@ class Hmis::Form::AssessmentDetail < ::GrdaWarehouseBase
   self.table_name = :hmis_assessment_details
   belongs_to :assessment, class_name: 'Hmis::Hud::Assessment'
   belongs_to :definition
+  belongs_to :processor, dependent: :destroy
+
+  def initialize(*)
+    super
+    build_processor
+  end
```

Comment:
Isn't this equivalent to:
```suggestion
  after_initialize :build_processor
```